### PR TITLE
Translate onboarding dialog to japanese

### PR DESCRIPTION
**Pull request by walf443**

### CODE REVIEW

File: app/assets/javascripts/components/locales/ja.jsx [hunk 1]
```diff
@@ -76,6 +76,24 @@ const ja = {
   "notifications.column_settings.show": "カラムに表示",
   "notifications.column_settings.sound": "通知音を再生",
   "notifications.settings": "カラム設定",
+  "onboarding.next": "次へ",
+  "onboarding.page_one.welcome": "Mastodonへようこそ！",
+  "onboarding.page_one.federation": "Mastodonは誰でも参加できるSNSです。",
+  "onboarding.page_one.handle": "あなたは今数あるMastodonインスタンスの1つである{domain}にいます。あなたのフルハンドルは{handle}です。",
```
Comment (usagi-f) [inline app/assets/javascripts/components/locales/ja.jsx]: 「フルハンドル」という言い回しはあまり馴染みがないです。
「ハンドル」もしくは「ハンドルネーム」が良いのではないでしょうか。

---

"フルハンドル" is not good.
"ハンドル" or "ハンドルネーム" is better.

Comment (Gargron) [inline app/assets/javascripts/components/locales/ja.jsx]: Should I wait with merging before this is cleared up?

Comment (walf443) [inline app/assets/javascripts/components/locales/ja.jsx]: "Full handle" should mean as global name on mastodon instances, I think. I agree to use "ハンドルネーム" more natural for japanese users. but that is not include that meanings. 

@usagi-f Do you have better idea for this?

Comment (walf443) [inline app/assets/javascripts/components/locales/ja.jsx]: @Gargron It's ok to merge this for the moment. And fix this point after the time.

**Review by Gargron — APPROVED** [review]

File: app/assets/javascripts/components/locales/ja.jsx (hunk 1 above)
Comment (usagi-f) [inline app/assets/javascripts/components/locales/ja.jsx]: @walf443 okay, i think so too

**Pull request merged**